recipes/libdmtx/all/patches/0002-0.7.7-add-install.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index a98e68f..0531d40 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -19,6 +19,9 @@ else()
+   target_link_libraries(dmtx PUBLIC -lm)
+ endif()
+ 
++set_target_properties(dmtx PROPERTIES PUBLIC_HEADER "${CMAKE_CURRENT_SOURCE_DIR}/dmtx.h")
```

Comment:
You should open a PR and share this with upstream! it's great work

Reply:
https://github.com/dmtx/libdmtx/pull/51/files
done